### PR TITLE
refactor private controller and corresponding queries

### PR DESCRIPTION
**Pull request by jay-meister**

### CODE REVIEW

**Comment by rjmk** [comment]
@JMurphyWeb Breaking tests

**Comment by rjmk** [comment]
@JMurphyWeb Conflicts!

**Comment by jay-meister** [comment]
I think this is ok now

File: api/controllers/PrivateController.js [hunk 1]
```diff
@@ -63,30 +80,10 @@ module.exports = {
...
-  },
-  sendSubsReminder: function (req, res) {
-    Members.query(queries.subscriptions, callback(res))
-  },
-  sendSubsReminderPost: function (req, res) {
-    Members.query(queries.newstype_post_nonzero, callback(res))
-  },
-  showMaintenance: function (req, res) {
-    res.view('pages/maintenance', {user: req.session.user})
+      [ dbCall('update_subscription')
```
Comment (rjmk) [inline api/controllers/PrivateController.js]: This needs to be the other way around (like in master)

**Comment by rjmk** [comment]
Mostly good. One important comment

**Pull request merged**